kitchen.recipes-config.yml

```diff
@@ -77,10 +77,10 @@ suites:
         - networking_configured
   - name: ssh_target_checker
```

Comment:
Please remove this test from here, as you already moved it to `platform-config` yaml.

Reply:
Will do thanks.